modules/task_class.js

```diff
@@ -0,0 +1,126 @@
+export default class Tasks {
+  constructor() {
+    if (!localStorage.getItem('tasks')) {
+      localStorage.setItem('tasks', JSON.stringify([]));
+    }
+    this.tasks = JSON.parse(localStorage.getItem('tasks'));
+  }
+
+  addTask(description) {
+    const task = { description, completed: false, id: this.tasks.length + 1 };
+    this.tasks.push(task);
+    localStorage.setItem('tasks', JSON.stringify(this.tasks));
+  }
+
+  removeTask(id) {
+    const index = this.tasks.findIndex((task) => task.id === Number(id));
+    if (index !== -1) {
+      this.tasks.splice(index, 1);
+      this.updateTaskIds();
+      localStorage.setItem('tasks', JSON.stringify(this.tasks));
+    }
+  }
+
+  updateTaskIds() {
+    this.tasks = this.tasks.map((task, index) => ({
+      ...task,
+      id: index + 1,
+    }));
+  }
+
+  editTask(id, newDescription) {
+    const index = this.tasks.findIndex((task) => task.id === Number(id));
+    if (index !== -1) {
+      this.tasks[index].description = newDescription;
+      localStorage.setItem('tasks', JSON.stringify(this.tasks));
+    }
+  }
+
+  updateTaskStatus(id, completed) {
+    const index = this.tasks.findIndex((task) => task.id === Number(id));
+    if (index !== -1) {
+      this.tasks[index].completed = completed;
+      localStorage.setItem('tasks', JSON.stringify(this.tasks));
+    }
+  }
+
+  displayTasks() {
```

Comment:
Your code's clean, maybe you have to consider putting some guide comments to make your code more readable and easy to understand for you or other developers.